frontend/packages/noobaa-storage-plugin/src/components/details-card/details-card.tsx

```diff
@@ -63,6 +63,9 @@ export const ObjectServiceDetailsCard: React.FC<DashboardItemProps> = ({
   const queryResult = prometheusResults.getIn([NOOBAA_SYSTEM_NAME_QUERY, 'result']);
 
   const systemName = _.get(queryResult, 'data.result[0].metric.system_name');
+  const systemAddress = _.get(queryResult, 'data.result[0].metric.system_address');
```

Comment:
Can we implement something like this:
```
  const noobaaSystemAddress = getMetric(queryResult, 'system_address');
  const noobaaSystemName = getMetric(queryResult, 'system_name');
  let link = null;
  if (noobaaSystemAddress && noobaaSystemName)
    link = `${noobaaSystemAddress}/fe/systems/${noobaaSystemName}/buckets/data-buckets`;
```
We don't need to pass then the `error` prop

Reply:
Ideally these lines should be wrapped in a function now, can do that in a followup PR

Reply:
@afreen23 will use the function for metric. However for the other link issue, I need to use the error prop because if link is undefined (which is passed through ExternalLink) then DetailItem will not figure out that the values are missing.